qa/rpc-tests/test_framework/mininode.py

```diff
@@ -36,7 +36,7 @@
 from threading import Thread
 import logging
 import copy
-import ltc_scrypt
+import litecoin_scrypt
```

Comment:
This kills CI.

https://travis-ci.com/github/dogecoin/dogecoin/jobs/481116840#L2375